### ISSUE TITLE
Can not select texts with EnableMouse(true)

### ISSUE THREAD

**Issue by rush1kesh-010** [rivo/tview#1027]
When we have `Application.EnableMouse(true)` , we can not select any text with mouse drag in the terminal.

Is it possible to use mouse events while allowing selection of texts with mouse drag?

Here's what I mean by selecting text using mouse drag:
![image](https://github.com/user-attachments/assets/5ec575f6-e083-46a4-94ea-9facbe5b2ac1)

**Comment by rivo** [comment]
It works for me in iTerm2 on macOS. Could it be that you need to enable it in your terminal software?

If your terminal software doesn't support selections while forwarding mouse events to the app running inside, there is not much `tview` can do about it. For example, `TextArea` and `InputField` do support text selections but it all happens inside the app. If you want to copy+paste this "emulated" selection like you're used to in non-mouse mode, it's not going to work because your terminal won't have any information about that selection. Unless, of course, your app has access to your local clipboard (which is possible but not very common because it's highly OS-dependent). I don't think this is something you intend to implement.

By the way, if your terminal supports it, you can add hyperlinks to the text, see [here](https://pkg.go.dev/github.com/rivo/tview#hdr-Styles__Colors__and_Hyperlinks). Your screenshot seemed like you might want to make those URLs clickable.

**Comment by rush1kesh-010** [comment]
Yes, holding shift bypasses the mouse event forwarding in GNOME terminal in ubuntu, and I can select text using this method.


But for Windows cmd and powershell, there is no bypassing supported for mouse event forwarding. I need to open the Window Control Menu and click `Edit->Mark` to enter text selection mode in Windows terminals. 


Is it possible to make the gnome terminal by default select text like MacOS?

**Comment by rivo** [comment]
I don't have an answer to this. As it is not really about `tview` but rather about the different terminal apps, I'm closing this issue.
